yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/commons/DockerCreateContainer.java

```diff
@@ -305,8 +311,22 @@ public String getCommand() {
     @Nonnull
     public String[] getDockerCommandArray() {
         String[] dockerCommandArray = new String[0];
+        final ArrayList<String> commands = new ArrayList<>();
         if (StringUtils.isNotEmpty(command)) {
-            dockerCommandArray = command.split(" ");
+
+            // https://stackoverflow.com/questions/3366281/tokenizing-a-string-but-ignoring-delimiters-within-quotes
+            String regex = "[\"\']([^\"]*)[\"\']|(\\S+)";
```

Comment:
It is wrong regexp. `"some 'very complex' command"` will be tokenized as `["some ", "very", "complex'", "command\""]`.